src/new-client/public/index.html

```diff
@@ -7,7 +7,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no"
     />
 
-
+    <link href="%PUBLIC_URL%/fontawesome/css/all.min.css" rel="stylesheet" />
```

Comment:
Font-awesome icons via css were removed so that we don't ship all of these webfonts to the browser (which will include all the icons we don't need).  We should be able to get all the icons we need from the React icons library that was added.  See this PR: https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1752